### PR TITLE
Tweak to bulk_update_metadata spec to allow for slow upload.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Test was failing due to slow upload.

Why is the upload slow? No idea.

### CODE REVIEW

File: spec/features/bulk_update_metadata_spec.rb [hunk 1]
```diff
@@ -28,6 +28,7 @@
     choose 'Spreadsheet input; load into objects'
     fill_in '3. Note', with: note
     click_button 'Submit'
+    sleep 5 # For some reason, somtimes this upload is slow.
```
Comment (jcoyne) [inline spec/features/bulk_update_metadata_spec.rb:31]: typo 'sometimes'

Comment (justinlittman) [reply spec/features/bulk_update_metadata_spec.rb]: No, nothing appears after the submit and the check below that involves a refresh interrupts the form submission. We're in some odd Capybara / browser twilight zone here. Open to suggestions.

And fixed typo.

Comment (jcoyne) [reply spec/features/bulk_update_metadata_spec.rb]: How about we add this first? https://github.com/sul-dlss/argo/pull/2715

Comment (justinlittman) [reply spec/features/bulk_update_metadata_spec.rb]: With that change, I was able to replace wait.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**